src/DotVVM.Samples.Common/ViewModels/FeatureSamples/ViewModelProtection/NestedSignaturesViewModel.cs

```diff
@@ -9,7 +9,7 @@ namespace DotVVM.Samples.Common.ViewModels.FeatureSamples.ViewModelProtection
     public class NestedSignaturesViewModel : DotvvmViewModelBase
     {
         [Protect(ProtectMode.SignData)]
-        public TypA ObjA { get; set; } = new TypA() { Next = new TypA() };
+        public TypeA ObjA { get; set; } = new TypA() { Next = new TypA() };
```

Comment:
This currently breaks build

```suggestion
        public TypeA ObjA { get; set; } = new TypeA() { Next = new TypeA() };
```